Sources/SashaCore/Services/IconService.swift

```diff
@@ -24,7 +24,7 @@ final class IconService {
         static let complicationSetName = "Complication.complicationset"
         static let contentsName = "Contents.json"
         static let androidIconFolder = "Android"
-        static let defaultSize: Float = 1024
+        static var defaultSize: Float = 0
     }
```

Comment:
I guess you can remove this variable and use only local variable

Reply:
Removed